### ISSUE TITLE
`Net::ZMQ` failing without `libzmq.so`

### ISSUE THREAD

**Issue by benjif** [perl6/ecosystem-unbitrot#318]
https://toast.perl6.party/module?module=Net::ZMQ&commit=2018.06

```
Cannot locate native library 'libzmq.so.5': libzmq.so.5: cannot open shared object file: No such file or directory
  in method setup at /home/cpan/toaster/build/2018_06/install/share/perl6/sources/24DD121B5B4774C04A7084827BFAD92199756E03 (NativeCall) line 283
```

**Comment by AlexDaniel** [comment]
This was the ticket for Net::ZMQ: https://github.com/perl6/ecosystem-unbitrot/issues/116

So the module is installable as long as the lib is installed, which is fine. [There is a way for a module to signal which native libs it needs](https://deathbyperl6.com/perl-toolchain-summit-2018-meta6-dependency-hello/), but during this squashathon it was decided not to enforce this yet (maybe next time).

In other words, the module is OK for now, and the ticket can be closed!
